### PR TITLE
Feat/purchase toggle

### PR DESCRIPTION
**Pull request by Amean**
updates the owner view page to include a toggle button for purchase, also arranges the side options to show even when drawer is collapsed and shows a price tag widget when NFT is for sale. Is a part of this issue: https://github.com/Pylons-tech/pylons/issues/1348

image

image

image

### CODE REVIEW

**Comment by Tanyaml** [comment]
@kjawadDeveloper2 please review.

**Comment by kjawadDeveloper2** [comment]
Kindly add screenshots

**Comment by kjawadDeveloper2** [comment]
@Amean  can you add a feature toggel?

File: wallet/lib/main_prod.dart [hunk 1]
```diff
@@ -17,6 +17,8 @@ import 'package:pylons_wallet/utils/constants.dart';
 import 'package:pylons_wallet/utils/dependency_injection/dependency_injection.dart' as di;
 import 'package:pylons_wallet/utils/dependency_injection/dependency_injection.dart';
 
+//ignore_for_file: deprecated_member_use
```
[the remark this answers is not in the document]
Comment (Amean) [reply wallet/lib/main_prod.dart]: the tests were failing with a deprecated error, after merging from main it is resolved so I removed it

**Pull request closed without merging**